zoo/auditing/assets/js/components/IssueList.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div class="issue-list ui segment loading">
```

Comment:
Check https://semantic-ui.com/modules/accordion.html

Reply:
I didn't use accordion in the end for the following reasons:

- it expects that the content is rendered just invisible, we would need to render all projects (which is a lot of html)
- it can still be done effectively, just render the content with vue and run `$(element).accordion('refresh')`, which I tried doing
- the problem is that the CSS is very different from `segments` and it looks weird, it would require a lot of changes (e.g. by default, accordion has max width `600 px`)
- I used `ui segments` and nested `ui segment` classes, it looks better and was much easier to do